tournaments/index.js

```diff
@@ -493,6 +497,28 @@ Tournament = (function () {
 		return true;
 	};
 
+	Tournament.prototype.setAutoStartTimeout = function (timeout, output) {
+		if (this.isTournamentStarted) {
+			output.sendReply('|tournament|error|AlreadyStarted');
+			return false;
+		}
+		if (timeout < AUTO_START_MINIMUM_TIMEOUT || isNaN(timeout)) {
+			output.sendReply('|tournament|error|InvalidAutoStartTimeout');
+			return false;
+		}
+
+		timeout = parseFloat(timeout);
```

Comment:
What's the advantage of calling it before? The autodq function does the same so I didn't think much of it.